adagrams/game.py

```diff
@@ -1,13 +1,162 @@
-from random import randint
+from random import randint 
+
+def create_letter_pool():
+    letter_pool = []
+    letter_pool.extend(["A"] * 9)
+    letter_pool.extend(["B"] * 2)
+    letter_pool.extend(["C"] * 2)
+    letter_pool.extend(["D"] * 4)
+    letter_pool.extend(["E"] * 12)
+    letter_pool.extend(["F"] * 2)
+    letter_pool.extend(["G"] * 3)
+    letter_pool.extend(["H"] * 2)
+    letter_pool.extend(["I"] * 9)
+    letter_pool.extend(["J"] * 1)
+    letter_pool.extend(["K"] * 1)
+    letter_pool.extend(["L"] * 4)
+    letter_pool.extend(["M"] * 2)
+    letter_pool.extend(["N"] * 6)
+    letter_pool.extend(["O"] * 8)
+    letter_pool.extend(["P"] * 2)
+    letter_pool.extend(["Q"] * 1)
+    letter_pool.extend(["R"] * 6)
+    letter_pool.extend(["S"] * 4)
+    letter_pool.extend(["T"] * 6)
+    letter_pool.extend(["U"] * 4)
+    letter_pool.extend(["V"] * 2)
+    letter_pool.extend(["W"] * 2)
+    letter_pool.extend(["X"] * 1)
+    letter_pool.extend(["Y"] * 2)
+    letter_pool.extend(["Z"] * 1)
+    return letter_pool
+
 
 def draw_letters():
-    pass
+    #Create the data structure containing the letters and letter distribution.
+    letter_pool = create_letter_pool()
+
+    #Randomly select 10 characters from the data structure
+    letter_hand = []
+    for i in range(10):
+        #Generates a random integer between 0 and the last element in letter_pool list.
+        random_index = randint(0,len(letter_pool)-1)
```

Comment:
Operators like `<` should be surrounded with white spaces. Take a look at the [PEP8 style guide on white spaces](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements).  In industry, code that doesn't follow an agreed upon code styles will need to be revised to ensure the codebase is neat and consistent.

```suggestion
        random_index = randint(0,len(letter_pool) - 1)
```